docs/src/components/productBaseUI/components/BaseMenuDemo.tsx

```diff
@@ -126,42 +128,45 @@ export default function BaseMenuDemo({ styling }: { styling?: 'system' | 'tailwi
       {styling === 'css' && <GlobalStyles styles={CSS} />}
       {styling === 'tailwindcss' && (
         <Dropdown>
-          <MenuButtonUnstyled className="min-h-[calc(1.5em + 22px)] inline-flex items-center gap-[0.5rem] rounded-[8px] border border-solid border-[--muidocs-palette-grey-200] bg-[--muidocs-palette-background-paper] p-[8px_12px_8px_6px] text-[0.875rem] leading-[1.5] transition-all [box-shadow:var(--shadow)] [font-family:IBM_Plex_sans] hover:bg-[--muidocs-palette-grey-50]  ui-focus-visible:[outline:3px_solid_var(--focus-ring)] dark:border-[--muidocs-palette-primaryDark-700] dark:hover:bg-[--muidocs-palette-primaryDark-800]">
+          <MenuButton
+            className="min-h-[calc(1.5em + 22px)] inline-flex items-center gap-[0.5rem] rounded-[8px] border border-solid border-[--muidocs-palette-grey-200] bg-[--muidocs-palette-background-paper] p-[8px_12px_8px_6px] text-[0.875rem] leading-[1.5] transition-all [box-shadow:var(--shadow)] font-['IBM_Plex_Sans'] hover:bg-[--muidocs-palette-grey-50] ui-focus-visible:[outline:4px_solid_var(--focus-ring)] 
```

Comment:
@michaldudak hey! 👋 Any idea why the dark declarations here are _not_ working on the Tailwind demos? It also happens to be the case in the Input demo.

Reply:
Just realized that it might be for the same reason as talked through on this issue, right? https://github.com/mui/material-ui/issues/38575#issuecomment-1838818935

Reply:
We may need to re-add `[data-mui-color-scheme="dark"]` to tailwind.config.js `darkMode`. It is buggy (as per #38575), but at least it works after a page refresh.

Reply:
Uhm... why did we remove it on https://github.com/mui/material-ui/pull/37960/? I can add it back, but want to ensure we're not breaking anything else. 😅

Reply:
Because I incorrectly thought Tailwind CSS is used only in component demos (which show usage of the `.dark` class name, not our custom data attribute).

Reply:
Got it! Alright, I'll put it back then 👍